definitions/tools/filter_sv_vcf_size.cwl

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: CommandLineTool
+
+baseCommand: ["/bin/bash", "filter_sv_vcf_size.sh"]
+requirements:
+    - class: ResourceRequirement
+      ramMin: 4000
+    - class: DockerRequirement
+      dockerPull: "mgibio/bcftools-cwl:1.9"
+    - class: InitialWorkDirRequirement
+      listing:
+      - entryname: "filter_sv_vcf_size.sh"
+        entry: |
+          #!/bin/bash
+          set -eou pipefail
+          
+          INPUT_VCF="$1"
+          OUTPUT_VCF="$2"
+          SV_SIZE="$3"
+          FILTER_METHOD="$4"
+
+          if [ "$FILTER_METHOD" == "max" ]; then
+            echo "Running filter for max size svs"
+            filter_expression="ABS(SVLEN) < $SV_SIZE"
+          elif [ "$FILTER_METHOD" ==  "min" ]; then
+            echo "Running filter for min size svs"
+            filter_expression="ABS(SVLEN) > $SV_SIZE"
```

Comment:
Do we want these to be inclusive ranges (i.e. `>=`)?  It might not make a huge difference either way; just not sure which way people would prefer to think about it.